microconfig/utils/src/test/java/io/microconfig/utils/StringUtilsTest.java

```diff
@@ -62,12 +83,38 @@ void testLike() {
         assertFalse(like("abc", "abc%_"));
         assertFalse(like("abc", "_%bc%_"));
         assertTrue(like("abc", "_%b%_"));
-        assertTrue(like("ru.sbt.risk.hmds.local.jdbc.password", "ru.sbt.risk.hmds.local%"));
+        assertTrue(like("ru.sbt.risk.hmds.local.jdbc.password",
+                "ru.sbt.risk.hmds.local%"));
+    }
+
+    @Test
+    void testSplitToList() {
+        ArrayList<String> list = new ArrayList<>();
+        list.add("f");
+        list.add("o");
+        list.add("o");
+        assertEquals(list, StringUtils.splitToList("f,o,o", ","));
     }
 
     @Test
     void testToLowerHyphen() {
         assertEquals("hello-world", toLowerHyphen("helloWorld"));
         assertEquals("yes", toLowerHyphen("yes"));
+        assertEquals("foo-bar-baz",
+                StringUtils.toLowerHyphen("fooBarBaz"));
+    }
+
+    @Test
+    void testIndexOfFirstDigitOr() {
+        assertEquals(2,
+                StringUtils.indexOfFirstDigitOr("foobar", 2));
+        assertEquals(5,
+                StringUtils.indexOfFirstDigitOr("fooba3r", 2));
+    }
+
+    @Test
+    void testAddOffsets() {
+        assertEquals("foo", StringUtils.addOffsets("foo", 0));
+        assertEquals("foo   ", StringUtils.addOffsets("foo", 3));
     }
 }
```

Comment:
Not your change but you could add a newline at the end of the file.

Reply:
Done :)